### ISSUE TITLE
Cassandra null value is presented as zero for numeric types in result view

### ISSUE THREAD

**Issue by arzt** [serge-rider/dbeaver#1153]
The result view of DBeaver EE should differentiate between `0` and `null` of numeric types in cassandra. Both values are represented by `0`.

Example:

```
CREATE KEYSPACE IF NOT EXISTS zeronull
WITH REPLICATION = { 'class' : 'SimpleStrategy', 'replication_factor' : 1 }

USE zeronull

CREATE TABLE test (
    id int,
    a int,
    b float,
    c double,
    PRIMARY KEY (id)
)

truncate test

INSERT INTO test (id, a, b, c) values (0, null, 0, 1.0)

INSERT INTO test (id, a, b, c) values (1, 0, null, 2.0)

INSERT INTO test (id, a, b, c) values (2, null, 0, null)

select * from test
```

DBeaver shows:

```
id |a |b |c |
---|--|--|--|
1  |0 |0 |2 |
0  |0 |0 |1 |
2  |0 |0 |0 |
```
cqlsh shows:
```
 id | a    | b    | c
----+------+------+------
  1 |    0 | null |    2
  0 | null |    0 |    1
  2 | null |    0 | null
```

**Comment by serge-rider** [comment]
Thanks for report

**Comment by alexhisen** [comment]
Note that Booleans display [NULL] correctly in Cassandra in DBeaver EE 4.0.5 but the 'Set to NULL' context-menu option has no effect.

**Comment by sbecainfo** [comment]
This also happens for numeric types in Snowflake in 4.0.7

**Comment by costinpapuc** [comment]
For DBeaver EE 5.2.2, the booleans are displayed as `false` instead of `null`

**Comment by serge-rider** [comment]
@costinpapuc thanks for the bug report. This bug will be fixed in the next version (5.3 hotfix actually)
